snooty/specparser.py

```diff
@@ -262,6 +262,46 @@ def create_role(self) -> Role:
         )
 
 
+@checked
+@dataclass
+class FacetDefinition:
+    name: str
+
+
+@checked
+@dataclass
+class TargetPlatformDefinition:
+    name: str
+    versions: List[FacetDefinition]
+
+@checked
+@dataclass
+class TaxonomySpec:
```

Comment:
One last series of requests: Could we move this and its dependent classes to a new file titled `taxonomy.py`?

Reply:
great call. moved to its own module